### DISCUSSION TITLE
How to add Custom Map for every game mode in rtv

### DISCUSSION THREAD

**Discussion by ravicant** [kus/cs2-modded-server#27] [Q&A]
How to add Custom Map for every game mode in rtv

**Comment by kus** [comment]
It’s only possible to add a map for every game mode for rtv (I think I need to build my own plugin).

Copy [my rtv file](https://github.com/kus/cs2-modded-server/blob/master/custom_files_example/addons/counterstrikesharp/plugins/RockTheVote/maps.txt) file and put it `custom_files/addons/counterstrikesharp/plugins/RockTheVote/maps.txt`
